sql/api/src/main/scala/org/apache/spark/sql/internal/types/GeographicSpatialReferenceSystemMapper.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.internal.types;
+
+/**
+ * Class for providing SRS mappings for geographic spatial reference systems.
+ */
+public class GeographicSpatialReferenceSystemMapper extends SpatialReferenceSystemMapper {
+  // Returns the string ID corresponding to the input SRID. If not supported, returns `null`.
+  public static String getStringId(int srid) {
+    return srsCache.isGeographicSrid(srid) ? srsCache.getSrsInfo(srid).stringId() : null;
```

Comment:
```suggestion
    SpatialReferenceSystemInformation srsInfo = srsCache.getSrsInfo(srid);
    return (srsInfo != null && srsInfo.isGeographic()) ? srsInfo.stringId() : null;
```
See comments for the `CartesianSpatialReferenceSystemMapper` class. Same idea here.